testdata/node/seccomp_nostat.json

```diff
@@ -0,0 +1,772 @@
+{
```

Comment:
@jhou1 I guess we do not need this seccomp profile. This has been encoded into base64 format and provided in machine config above.

Reply:
Removed all files that are not referecened